doc-surrealdb_versioned_docs/version-latest/introduction/concepts/namespace.mdx

```diff
@@ -20,7 +23,30 @@ Namespaces are defined using the [`DEFINE NAMESPACE`](/docs/surrealdb/surrealql/
 DEFINE NAMESPACE [ OVERWRITE | IF NOT EXISTS ] @name [ COMMENT @string ]
 ```
 
-You can also view the list of namespaces in your SurrealDB instance using the [`INFO`](/docs/surrealdb/surrealql/statements/info) statement. This statement provides information about the accesses, databases, and users present in the current SurrealDB instance.
+Once a namespace has been defined the command below, you will get a list of all the namespaces in your SurrealDB instance. The output will include the namespace name and a list of users.
```

Comment:
has been defined the command below -> defined, the command below will display a list of...

list of users -> list of namespace users

(Since there are also database and root users)

Reply:
on it